extensions/adobe/experience/customerJourneyManagement/messageprofile.schema.json

```diff
@@ -37,6 +37,16 @@
           "meta:titleId": "messageprofile##https://ns.adobe.com/experience/customerJourneyManagement/messageProfile/variant##title##71711",
           "meta:descriptionId": "messageprofile##https://ns.adobe.com/experience/customerJourneyManagement/messageProfile/variant##description##28141"
         },
+        "https://ns.adobe.com/experience/customerJourneyManagement/messageProfile/isExecutionSendTimeOptimized": {
+          "title": "isExecutionSendTimeOptimized",
+          "type": "boolean",
+          "description": "True if the message execution is send time optimized."
+        },
+        "https://ns.adobe.com/experience/customerJourneyManagement/messageProfile/isTestProfile": {
+          "title": "isTestProfile",
+          "type": "boolean",
+          "description": "True if the message is sent as Test Profile."
```

Comment:
Should we rename `if the message is sent as Test Profile` to `if the message is sent to a Test Profile`